### PR TITLE
Added disabled option for Nav, Dropdown and Tabs

### PR DESCRIPTION
**Pull request by Thoulah**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | -

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Yii Framework 2 bootstrap4 extension Change Log
 
 - Bug #131: Fixed element with role tab must be inside role tablist (simialbi)
 - Enh #129: Added `Modal::SIZE_EXTRA_LARGE` constant (shoomlix)
+- Enh: Added disabled option for Nav, Dropdown and Tabs (Thoulah)
```
Comment (samdark) [inline CHANGELOG.md]: `#137`

File: src/Dropdown.php [hunk 2]
```diff
@@ -112,7 +113,15 @@ protected function renderItems($items, $options = [])
             $itemOptions = ArrayHelper::getValue($item, 'options', []);
             $linkOptions = ArrayHelper::getValue($item, 'linkOptions', []);
 //            $linkOptions['tabindex'] = '-1';
```
Comment (samdark) [inline src/Dropdown.php]: That is not about this pull request but... any idea why this commented out line is there?

Comment (Thoulah) [reply src/Dropdown.php]: I was wondering the same

Comment (Thoulah) [reply src/Dropdown.php]: Changed in https://github.com/yiisoft/yii2-bootstrap4/commit/61ddd1bcbd7381d734b756b48ca26a147e9bf1ec

**Comment by Thoulah** [comment]
Should DropdownTest have additional tests?

File: src/Dropdown.php (hunk 2 above)
Comment (samdark) [reply src/Dropdown.php]: I think we may safely delete it.

Comment (Thoulah) [reply src/Dropdown.php]: `$itemOptions` is also unsued, or am I overlooking anything?

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**